docs/stepactions.md

```diff
@@ -116,9 +116,9 @@ When referencing a `StepAction`, a `Step` can also provide it with `params`, jus
 
 ```yaml
 apiVersion: tekton.dev/v1
-kind: Task
+kind: TaskRun
```

Comment:
As is it will still fail, as the `TaskRun` doesn't specify the "required" parameter. I think it was meant to be a `Task` without the `taskSpec` part.